docs/content/middlewares/addprefix.md

```diff
@@ -26,6 +26,20 @@ The AddPrefix middleware updates the URL Path of the request before forwarding i
           - "traefik.http.middlewares.add-bar.addprefix.prefix=/bar"
     ```
 
+??? example "Kubernetes -- Prefixing with /bar"
+
+    ```yaml
+    apiVersion: traefik.containo.us/v1alpha1
+    kind: Middleware
+    metadata:
+      name: addprefix
+      namespace: default
```

Comment:
Don't need to define a namespace if it is the default

Reply:
Sure. but do you think it harms readability or clarity to keep it? And if not, do you suggest removing it in all other places as well?

Reply:
mmmm Not sure. I'm ok keeping it, and revisiting at a later date :)

Reply:
Ok, since they were all using "default", I've removed all instances of namespaces, for the sake of clarity. I guess we can re-add some when we have specific examples of dedicated namespace usage.